clippy_lints/src/from_over_into.rs

```diff
@@ -181,9 +181,6 @@ fn convert_to_from(
     let from = self_ty.span.get_source_text(cx)?;
     let into = target_ty.span.get_source_text(cx)?;
 
-    let return_type = matches!(sig.decl.output, FnRetTy::Return(_))
```

Comment:
I noticed that this didn't get synced (together with a few other things) when doing the Josh setup. With Josh all of that would be synced. The problem with `subtree` is, that if you mess up a merge once, it will never be synced correctly again. Luckily this didn't happen that often yet and will get fixed, once we move to Josh.

This is a commit fixing everything that wasn't correctly synced with `subtree`: https://github.com/flip1995/rust-clippy/pull/8/commits/89f11457f0980422c32ae966f34d5c5ca89470e3